eos/rare-b-decays/b-to-kstar-gamma-bfs2004_TEST.cc

```diff
@@ -63,10 +63,24 @@ class BToKstarGammaTest :
                 p["b->s::Re{c7'}"] = -0.00659; // m_s(m_b) / m_b(m_b) * Abs{c7} = 85 / 4200 * Abs{c7}
                 p["b->s::c8"] = -0.181;
                 // PDG 2010 CKM parameters
-                p["CKM::A"] = 0.812;
-                p["CKM::lambda"] = 0.22543;
-                p["CKM::rhobar"] = 0.144;
-                p["CKM::etabar"] = 0.342;
+                p["CKM::A"]         =  0.812;
+                p["CKM::lambda"]    =  0.22543;
+                p["CKM::rhobar"]    =  0.144;
+                p["CKM::etabar"]    =  0.342;
+                p["CKM::abs(V_ub)"] =  0.003540950873054711;
+                p["CKM::arg(V_ub)"] = -1.1728563751359748;
+                p["CKM::abs(V_cb)"] =  0.04126451344307112;
+                p["CKM::arg(V_cb)"] =  0.0;
+                p["CKM::abs(V_tb)"] =  0.9991419776905534;
+                p["CKM::arg(V_tb)"] =  0.0;
+                p["CKM::abs(V_td)"] =  0.008576901910577167;
```

Comment:
@dvandyk how did you compute those values? Taking the above A, lambda, etabar and rhobar, I find (small) inconsistencies.

Reply:
I used the EOS python interface, set A, lambda, rhobar, and etabar, and evaluated. Basically as follows:
```
import eos, cmath
p = eos.Parameters()
p['CKM::A'].set(...)
... # ditto for the rest
m = eos.Model.make('SM', p, eos.Options())
display(abs(m.ckm_ub()), cmath.phase(m.ckm_ub()))
... # ditto for the remaining CKM matrix elements
```

Reply:
Ok, the difference that I have are probably negligible (there are mostly on the arguments).